nixos/modules/programs/ssmtp.nix

```diff
@@ -21,9 +21,11 @@ in
     (mkRenamedOptionModule [ "networking" "defaultMailServer" "useTLS" ] [ "services" "ssmtp" "useTLS" ])
     (mkRenamedOptionModule [ "networking" "defaultMailServer" "useSTARTTLS" ] [ "services" "ssmtp" "useSTARTTLS" ])
     (mkRenamedOptionModule [ "networking" "defaultMailServer" "authUser" ] [ "services" "ssmtp" "authUser" ])
-    (mkRenamedOptionModule [ "networking" "defaultMailServer" "authPass" ] [ "services" "ssmtp" "authPass" ])
     (mkRenamedOptionModule [ "networking" "defaultMailServer" "authPassFile" ] [ "services" "ssmtp" "authPassFile" ])
     (mkRenamedOptionModule [ "networking" "defaultMailServer" "setSendmail" ] [ "services" "ssmtp" "setSendmail" ])
+
+    (mkRemovedOptionModule [ "networking" "defaultMailServer" "authPass" ] "Use services.ssmtp.authPassFile instead")
+    (mkRemovedOptionModule [ "services" "ssmtp" "authPass" ] "Use services.ssmtp.authPassFile instead")
```

Comment:
A reasoning why (I know, because of credentials leaked to the store) may be helpful. I'm not sure if everyone who stumbles upon this, is aware of this :)

Reply:
Oh, I meant a short reasoning in the error-message :) I'm well-aware of that :p

Reply:
Oh... :smile: Any suggestions? Something along the lines of "Use services.ssmtp.authPassFile instead to avoid leaking this password into the world-readable nix store"?

Reply:
`authPass has been removed since it leaks the clear-text password into the world-readable store. Use authPassFile instead and make sure it's not a store path`.

Just mentioning this to make sure that we spread the knowledge about this :)

Reply:
No doubt that is better. Thanks!